app/scripts/templates/sign_up.mustache

```diff
@@ -19,8 +19,13 @@
     {{^error}}
     <div class="error"></div>
     <div class="success"></div>
+
+    {{#showSyncSuggestion}}
+      <div class="info nudge" id="suggest-sync">{{#t}}Looking for Firefox Sync? <a href="https://mozilla.org/firefox/sync">Get started here.</a>{{/t}}</div>
+    {{/showSyncSuggestion}}
+    
     {{#isSyncMigration}}
-      <div class="info nudge">{{#t}}Migrate your sync data by creating a new Firefox&nbsp;Account.{{/t}}</div>
+      <div class="info nudge migration">{{#t}}Migrate your sync data by creating a new Firefox&nbsp;Account.{{/t}}</div>
```

Comment:
extending from https://github.com/mozilla/fxa-content-server/pull/3870/files#r68660957

I'd say let's make this consistent and go with something like `.info-suggest-sync` and `.info-migration` classes for these `info` panels.